client/js/app/package.json

```diff
@@ -35,7 +34,6 @@
     "husky": "^7",
     "lodash": "^4",
     "prettier": "2",
-    "pretty-quick": "^3",
```

Comment:
This is used by `husky` in `pre-commit`:

`npx pretty-quick --staged`

Reply:
Thanks, added back.